.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+#### Ticket (For Vimeo staff only)
+
+[TICKET_NUMBER](https://vimean.atlassian.net/browse/TICKET_NUMBER)
+
+#### Issue Summary
+
+- *Outline the goals of the PR in technical terms. If this branch fixes a bug, state what caused the bug.*
+
+#### Pull Request Checklist
```

Comment:
I think it makes sense for `Issue Summary` to immediately precede `Implementation Summary`. `Pull Request Checklist` may make sense to move up or down. I almost feel like moving it up would be better, so reviewers immediately know the state of the PR before investing time reading summaries. What do you think?